src/convertToObject.js

```diff
@@ -1,12 +1,22 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredArr = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item);
+
+  const newArr = filteredArr
```

Comment:
The variable name 'newArr' does not clearly describe the contents of the array. Consider using a more descriptive name, such as 'styleEntries'.